packages/server/src/controllers/accounts.ts

```diff
@@ -41,42 +62,45 @@ export const getCurrentAccount = async (ctx: Context) => {
     const sessionId = extractSessionId(ctx);
 
     if (!sessionId) {
-      ctx.body = { error: "Unauthorized", error_description: "no_session_id" };
       ctx.status = 401;
+      ctx.message = "Unauthorized - No sessionId";
       return;
     }
 
     const isAuthenticated = await verifySession(sessionId);
 
     if (!isAuthenticated) {
-      ctx.body = {
-        error: "Unauthorized",
-        error_description: "invalid_session",
-      };
       ctx.status = 401;
+      ctx.message = "Unauthorized - Session no longer active";
       return;
     }
 
     const user = await retrieveAccountBySessionId(sessionId);
 
     if (!user) {
-      ctx.body = { error: "Server error", error_description: "no_user_found" };
+      const errorMessage = "Unauthorized - User linked to session not found";
+      logger.error(errorMessage);
       ctx.status = 500;
+      ctx.message = errorMessage;
       return;
     }
 
+    ctx.status = 200;
+    ctx.message = "User authenticated";
     ctx.body = {
       user: {
         firstName: user.first_name,
         lastName: user.last_name,
       },
     };
-    ctx.status = 200;
-    ctx.message = "User authenticated";
   } catch (error) {
-    console.log(error);
-    console.log("Error when fetching current account");
+    const errorMessage = "Error when fetching current account";
+
+    logger.error(errorMessage, {
+      error: sanitizeError(error),
+    });
```

Comment:
we could wrap the logger.error function so that we are not forced to call sanitizeError everywhere..

Reply:
I create a ticket to do this with the global logger of winston :))